app/assets/javascripts/editor.js

```diff
@@ -676,6 +682,7 @@ $(function() {
   };
 
   var isBrowserSupported = function() {
+    // todo event streams are no longer required with websockets
     return window.EventSource !== undefined;
```

Comment:
Maybe we should check if the browser supports websockets, this could be done by https://modernizr.com/docs